contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -2301,24 +2301,24 @@ pltsql_report_proc_not_found_error(List *names, List *fargs, List *given_argname
 	{
 		const char *arg_str = (max_nargs < 2) ? "argument" : "arguments";
 
+		if (!proc_call)
```

Comment:
Why did we redefine the code block here?

Reply:
This is because in case of CONCAT() and CONCAT_WS() since we use variadic function argument, the no. of arguments can mismatch with the definition, resulting in `found` to be false, but we need to run our special function handling in that case as well to throw appropriate error message, so moved it out of that block.